Frontend/ui-driver/src/Screens/HomeScreen/Controller.purs

```diff
@@ -246,7 +248,8 @@ data Action = NoAction
 eval :: Action -> ST.HomeScreenState -> Eval Action ScreenOutput ST.HomeScreenState
 
 eval AfterRender state = do
-  continue state{props{mapRendered= true}}
+  updateAvailable <- pure $ checkUpdate unit
```

Comment:
use like `let  updateAvailable =  checkUpdate unit` try to reduce usage of `pure`

Reply:
Done